test-cases/nemesis/longevity-5gb-1h-CreateIndexNemesis.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 90
```

Comment:
Coll that you remembered to add it!
(We will need to remember this for every nemesis we add).

Reply:
I didn't remember... this is automatically generated file :sweat_smile:

Reply:
we should be checking it somehow, in a unittest or a new step for the CI

Reply:
oh, you meant the file :) yes, somehow I remembered